src/Illuminate/Auth/Console/stubs/make/views/layouts/app.stub

```diff
@@ -15,9 +15,9 @@
 
     <!-- Scripts -->
     <script>
-        window.Laravel = {!! json_encode([
-            'csrfToken' => csrf_token(),
-        ]) !!};
+        window.Laravel = {
+            csrfToken: document.querySelector('meta[name="csrf-token"]').getAttribute('content')
```

Comment:
Because by calling csrf_token() helper in both meta tag and script tag makes twice the work and because it's more elegant.